src/content/docs/browser/browser-monitoring/browser-pro-features/auto-logging.mdx

```diff
@@ -0,0 +1,78 @@
+---
+title: "Auto logging"
+metaDescription: "Get a more complete picture of your front-end application"
+freshnessValidatedDate: 2024-11-18
+---
+
+Auto logging collects log messages issued from the browser `console` to help you maximize observability of your front-end applications.
+
+Browser logs are tracked by default for Pro and Pro+SPA agents, but unavailable for the Lite browser agent. We recommend that you first confirm you're using the Pro or Pro+SPA agent.  See [Getting Started](#get-started).
+
+## How browser auto-logging works [#how-it-works]
+
+Based on logging levels and sampling rates set in the configuration, auto-logging may collect messages from the following:
+- `console.error`
+- `console.warn`
+- `console.info`
+- `console.log` (this will be treated the same as `INFO` level)
+- `console.debug`
+- `console.trace`
+
+Data passed through the console methods may go through deserialization and obfuscation.  Depending on the size and frequency, this may negatively impact application performance as well as data costs.  In general, it is NOT recommended to pass in large objects or large amounts of data into console methods.
+
+By default, browser event data is stored for 8 days, but actual data retention depends on your account.
```

Comment:
This stores `log` data, not browser data.  May need to check what the retention is for that

Reply:
Fixed, good call!